browser/ui/views/infobars/brave_wayback_machine_infobar_contents_view.cc

```diff
@@ -42,6 +43,8 @@
 #include "ui/views/view_class_properties.h"
 #include "url/gurl.h"
 
+static_assert(!BUILDFLAG(IS_ANDROID));
```

Comment:
Just curious this assertion is necessary because all files in `b/b/ui/views/` are desktop only files.

Reply:
Initial intention of `static_assert(!BUILDFLAG(IS_ANDROID));` was to ensure that files which do either
```
#include "brave/grit/brave_theme_resources.h"
```
or
```
#include "brave/grit/brave_theme_resources_map.h"
```
are not compiled on Android.

But now I excluded generation of `brave_theme_resources.h`/`brave_theme_resources_map.h` headers, so `static_assert(!BUILDFLAG(IS_ANDROID))` is unnecessary.
Removed that at https://github.com/brave/brave-core/compare/f9ef3a824839e9907b9bed0bab0aec0f39b731c6..ad4d7db397c81d8cb1e121daddf1e573553f44d4 and https://github.com/brave/brave-core/compare/ad4d7db397c81d8cb1e121daddf1e573553f44d4..25d376ccffb2a4c33c8b85f7b9f68123fd14a8f0.

Thanks for this notice :+1: .